test/dummy/config/environments/production.rb

```diff
@@ -82,8 +84,8 @@
   # require "syslog/logger"
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
 
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
+  if ENV['RAILS_LOG_TO_STDOUT'].present?
+    logger           = ActiveSupport::Logger.new($stdout)
```

Comment:
@GowthamK01 Please double check this change once

Reply:
@gomathi-mallow Checked and reverted the change as it is a global variable than a constant